specification/cdn/resource-manager/readme.md

```diff
@@ -25,8 +25,25 @@ To see additional help and options, run:
 These are the global settings for the Cdn API.
 
 ``` yaml
+title: CdnManagementClient
+description: Cdn Management Client
 openapi-type: arm
-tag: package-2019-04
+tag: package-2019-06
+```
+
+### Tag: package-2019-06
+
+These settings apply only when `--tag=package-2019-06` is specified on the command line.
+
+``` yaml $(tag) == 'package-2019-06'
+input-file:
+- Microsoft.Cdn/stable/2019-06-15/cdn.json
```

Comment:
are you intentionally combining a stable and preview version of APIs in your latest package? This is the package that will be used to ship SDKs from by default.